extensions/mssql/src/objectManagement/objectManagementService.ts

```diff
@@ -432,6 +434,40 @@ export class TestObjectManagementService implements IObjectManagementService {
 		};
 	}
 
+	private getDatabasePropertiesView(isNewObject: boolean, name: string): ObjectManagement.DatabasePropertiesViewInfo {
+		return isNewObject ? <ObjectManagement.DatabasePropertiesViewInfo>{
```

Comment:
Is there any way we're able to get here if this is a new object? Would it be better to throw an error in that case?

Reply:
For properties we always not a new object specific. I dont think we need this one at all, unless we use the same viewInfo for the new database object, but if needed we can add this later.